content/blog/newsletter_2022-02-05.md

```diff
@@ -0,0 +1,236 @@
+
+There are several articles to the topic  "messenger" at the German site from "[Freie Messenger](https://www.freie-messenger.de/#neuigkeiten-letzte-änderungen)" with focus on alternatives to WhatsApp, E2EE, interoperability, security/pseudosecurity. You may help translating them to your native language.
+
+[OMEMO was finally integrated in Movim](https://mov.im/?post/pubsub.movim.eu/Movim/end-to-end-encryption-in-movim-omemo-is-finally-there-yudZPP) after 6 long years of discussions. In this article we are explaining what is the general OMEMO architecture what were the difficulties encountered when doing the integration in Movim and how we overcame them.
+
+As the [previously announced](https://snikket.org/blog/simply-secure-collaboration/) collaboration between Snikket and Simply Secure ended its first project, they interviewed the project's founder, Matthew Wild, about Snikket's origins and his experience managing open-source projects. Read the interview: [On Getting Things Done: A Conversation with Matthew Wild from Snikket](https://simplysecure.org/blog/on-getting-things-done-a-conversation-with-matthew-wild-from-snikket/).
+
+Mellium Co-op has published their [Year in Review for 2021](https://opencollective.com/mellium/updates/mellium-year-in-review-2021) and the [Dev Communiqué for December 2021](https://opencollective.com/mellium/updates/dev-communique-for-december-2021).
+
+MongooseIM writes about [Dynamic XMPP Domains in their solutions](https://www.erlang-solutions.com/blog/dynamic-xmpp-domains-in-mongooseim/)
+
+XMPP passes in a [congested network](https://blog.lewman.com/internet-messaging-versus-congested-network.html).
+
+A short introduction to the [Quicksy Android client](https://ravidwivedi.in/posts/quicksy-app/index.html)
+
+The German Linux Magazin has been [testing free instant messaging clients](https://www.linux-magazin.de/ausgaben/2022/02/bitparade/) for Linux in their latest printout and nearby other messenger, reviewed the [Gajim desktop client](https://gajim.org/).
+
+A critic on [XMPP server security](https://bishopfox.com/blog/xmpp-underappreciated-attack-surface
+) by Bishop Fox.
+
+
+## Software news
+
+### Clients and applications
+
+[Gajim development news](https://gajim.org/post/2022-01-22-development-news-december-january/): Work on Gajim 1.4 is making big steps forward! After nine months of developing Gajim’s new main window, the code was finally ready to be merged into the master branch. This enables automatic builds of nightly versions for Linux and Windows.
+
+[monocles chat](https://f-droid.org/de/packages/de.monocles.chat/) (a fork of Conversations and Blabber.im) will get OTR in the next release. The client also only allows connections to XMPP servers with up to date SSL configurations and does not offer fallback SSL connections to avoid data leaks. Nevertheless it is compatible with every current XMPP account.
```

Comment:
```suggestion
[monocles chat](https://f-droid.org/de/packages/de.monocles.chat/) (a fork of Conversations and Blabber.im) will get OTR support in the next release. The client also only allows connections to XMPP servers with up to date SSL configurations and does not offer fallback SSL connections to avoid data leaks. Nevertheless it is compatible with every current XMPP account.
```